cmd/admin.go

```diff
@@ -355,6 +355,10 @@ func runRepoSyncReleases(_ *cli.Context) error {
 		return err
 	}
 
+	if err := git.InitSimple(ctx); err != nil {
```

Comment:
I think this function should short-circuit if it is already initialized.
So, wrap its content in a `once.Do`?